src/fps/addresses/1.toml

```diff
@@ -0,0 +1,4 @@
+[[addresses]]
+addr = "0x240B3bd6b95cE40497Aafd71aD4705d0345A33CD"
+identifier = "SYSTEM_CONFIG_LOGIC"
+isContract = true
```

Comment:
In the spirit of avoiding hardcoding addresses as much as possible, do we have to hardcode these logic addresses? They can be read onchain so we should be able to delete this. We often upgrade the SystemConfig logic address, and now this file becomes outdated